atlasdb-config/src/main/java/com/palantir/atlasdb/http/ProtocolAwareExceptionMapper.java

```diff
@@ -49,6 +53,7 @@ public Response toResponse(E exception) {
         AtlasDbHttpProtocolVersion version = getHttpProtocolVersion();
         switch (version) {
             case LEGACY_OR_UNKNOWN:
+                logLegacyProtocolUsageIfPossible();
```

Comment:
can/should we ping a metric here?

Reply:
We could, though yeah the main reason for doing this is to avoid cost. One gauge would be a few hundred dollars per month - admittedly not a lot, but this isn't something I'd generally need to know on short notice

Reply:
right then - makes sense not to include a metric here.